plugin/trino-mysql-event-listener/src/main/java/io/trino/plugin/eventlistener/mysql/MysqlEventListenerConfig.java

```diff
@@ -49,4 +51,17 @@ public boolean isValidUrl()
             throw new RuntimeException(e);
         }
     }
+
+    public boolean getTerminateOnInitializationFailure()
+    {
+        return terminateOnInitializationFailure;
+    }
+
+    @ConfigDescription("MySQL event listener initialization might fail due to network issues reaching the MySQL server. This flag controls whether to throw an exception in such cases.")
```

Comment:
The initialization fails for various reasons. No need to mention "network" here.